kwanj-k/store_manager
# Add a static store manager UI

### What does this PR do?
Adds the the static version of the site

#### Description of Task to be Completed ?
-- Have a static version of the site to show how the UI will look

### How should this be manually tested?
After cloning the repo,git pull the branch UI

Then open the index.html file and check out the different pages

## Screenshots
* Landing Page
![sml](https://user-images.githubusercontent.com/33450849/47313716-53664200-d648-11e8-929f-c210a4baf149.png)

* Products Page
![smp](https://user-images.githubusercontent.com/33450849/47313736-60833100-d648-11e8-8ead-5ab36bd46545.png)
